open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/cache/impl/DataElementCacheImpl.java

```diff
@@ -644,9 +644,10 @@ public void query(CacheQuery query)
      * Removes the Identified IDs from the cache.
      *
      * @param ids the IDs to remove from the cache.
+     * @param fullClear whether it's a full clear for the layer

```

Comment:
not sure of the formatting for @param tags with periods but some have them some dont